### DISCUSSION TITLE
Traiter le cas des artistes-auteurs à la retraite

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2734] [Idées]
Les questions concernant les artistes-auteurs sont orientées pour une personne active qui n'est pas retraitée. Il faudrait soit les réécrire pour inclure ce public, ou demander en amont si la personne est à la retraite ou non.

**Comment by Juliedelevoie** [comment]
Pris en compte du statut de retraité : "si l'intéressé est déjà retraité doit-il encore verser une cotisation à l'URSSAF? ce critère modifie t-il la simulation?"
